MIGRATING_to_v3.md

```diff
@@ -0,0 +1,70 @@
+# Migrate to Design System version 3
+
+## Benefits
+
+- Tree-shaking for the `core` package (`pcln-design-system`)
+- Tree-shaking for the `icons` package (`pcln-icons`)
+  - Can now use `import { TrendingUp, Timer } from 'pcln-icons'` import syntax
+- Tree-shaking for the `autocomplete` package (`pcln-autocomplete`)
+- Tree-shaking for the `modal` package (`pcln-modal`)
+- Tree-shaking for the `popover` package (`pcln-popover`)
+- Tree-shaking for the `slider` package (`pcln-slider`)
+
+## Breaking Changes
+
+### Core (`pcln-design-system`)
+
+- peerDependency increase: `pcln-icons >=3`
+- GreenButton and RedButton have been removed
+  - Use `<Button color='secondary'>...</Button` or `<Button color='error'>...</Button>` instead
+- OutlineButton `<OutlineButton />` has been removed
+  - Use `<Button variation='outline'>...</Button>` instead
+- `bg` prop has been marked as deprecated in favor of `color`
+- Icon `<Icon name="iconName" />` has been removed
+- IconButton
+  - `name`, `size` and `color` props are removed
+  - added `icon` prop of type node. You should pass the `size` and `color` props directly to the icon child:
+
+```
+import { IconButton } from 'pcln-design-system
+import { ThumbsUp } from 'pcln-icons
+
+<IconButton
+    icon={<ThumbsUp size={32} color='primary' />}
+/>
+```
+
+- Components that now take an `icon` node instead of an `iconName` string
+
+  - Icon
+  - Hug
+  - Banner
+  - Step
+  - Select
```

Comment:
Passing in an icon node should also be possible in v2 for most of these components, otherwise I'm not sure v3 will be backward compatible with v2.

Reply:
I think IconButton can be added to this list as well

Reply:
The goal isn't for it to be backwards compatible. Unfortunately some of the changes required were breaking for v2, which is why this needed to be a v3 bump and couldn't be a bunch of minor bumps.

Reply:
You should definitely be able to pass an icon node.